src/go/configgenerator/listener_generator.go

```diff
@@ -49,40 +49,33 @@ func MakeListeners(serviceInfo *sc.ServiceInfo) ([]*listenerpb.Listener, error)
 	return []*listenerpb.Listener{listener}, nil
 }
 
-// AddPerRouteConfigGenToMethods adds the filterGenerator functions to all the methods in place.
-func AddPerRouteConfigGenToMethods(methods []*sc.MethodInfo, filterGen FilterGenerator) error {
-	for _, method := range methods {
-		method.PerRouteConfigGens = append(method.PerRouteConfigGens, &sc.PerRouteConfigGenerator{
-			FilterName:            filterGen.FilterName(),
-			PerRouteConfigGenFunc: filterGen.GenPerRouteConfig,
-		})
-	}
-	return nil
-
-}
-
 // GetFilterConfigAndAddPerRouteConfigGen does two things
 //   - Return all http filter configs in a list
 //   - In place add the perRouteConfigGen function to all methods defined in serviceInfo
 func GetFilterConfigAndAddPerRouteConfigGen(serviceInfo *sc.ServiceInfo, filterGenerators []FilterGenerator) ([]*hcmpb.HttpFilter, error) {
```

Comment:
Rename to `MakeHttpFilterConfigs`?
Remove `- In place add the perRouteConfigGen function to all methods defined in serviceInfo`

Reply:
Done